snapcraft/tests/test_lxd.py

```diff
@@ -165,15 +165,15 @@ def run_effect(*args, **kwargs):
 
     @patch('snapcraft.internal.lxd.Cleanbuilder._container_run')
     def test_lxc_check_fails(self, mock_run):
-        self.useFixture(tests.fixture_setup.FakeLXD(fail_on_default=True))
+        fake_lxd = tests.fixture_setup.FakeLXD()
+        self.useFixture(fake_lxd)
+        fake_lxd.check_output_mock.side_effect = FileNotFoundError('lxc')
```

Comment:
There is a coincidence today that the first `check_call` or `check_output` call is for `lxc` but to be complete and future proof we should really be checking if `args[0]` includes `lxc` in the command and if not, not raise the exception.